### PR TITLE
Fix errors in renaming state to stage for workflow

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Revert erroneous changing the column name in sql update file https://github.com/joomla/joomla-cms/pull/21544/files#diff-1df6837d39e992f5ee0904c00cbfe685R81
Fix error in Workflow Table Stage

### CODE REVIEW

File: administrator/components/com_workflow/Table/StageTable.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ public function delete($pk = null)
 		{
 			$query = $db->getQuery(true)
 				->delete($db->quoteName('#__workflow_transitions'))
-				->where($db->quoteName('to_') . ' = ' . (int) $pk, 'OR')
+				->where($db->quoteName('to_stage') . ' = ' . (int) $pk, 'OR')
```
Comment (richard67) [inline administrator/components/com_workflow/Table/StageTable.php]: This should be `to_stage_id` and not `to_stage` as far as I can see in the installation joomla.sql for mysql.

**Review by richard67 — CHANGES_REQUESTED** [review]
In file administrator/components/com_workflow/Table/StageTable.php, line 81, it should be `to_stage_id` and not `to_stage` as far as I can see in the installation joomla.sql for mysql.

**Comment by richard67** [comment]
I have tested this item :red_circle: unsuccessfully on 195851bfa15835d1043a9bee4e727f429f3ad0f7<br /><br />In file administrator/components/com_workflow/Table/StageTable.php, line 81, it should be to_stage_id and not to_stage as far as I can see in the installation joomla.sql for mysql.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21563">issues.joomla.org/tracker/joomla-cms/21563</a>.</sub>

File: administrator/components/com_workflow/Table/StageTable.php (hunk 1 above)
Comment (chmst) [reply administrator/components/com_workflow/Table/StageTable.php]: thanks, you are right

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 124473e9a652ae39006cdb778da84a9006278baf<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21563">issues.joomla.org/tracker/joomla-cms/21563</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 124473e9a652ae39006cdb778da84a9006278baf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21563">issues.joomla.org/tracker/joomla-cms/21563</a>.</sub>

**Comment by infograf768** [comment]
RTC @wilsonge @laoneo <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21563">issues.joomla.org/tracker/joomla-cms/21563</a>.</sub>

**Pull request merged**